audio/src/main/java/edu/artic/audio/AudioDetailsViewModel.kt

```diff
@@ -3,24 +3,37 @@ package edu.artic.audio
 import com.fuzz.rx.bindTo
 import com.fuzz.rx.disposedBy
 import edu.artic.db.models.ArticObject
+import edu.artic.db.models.AudioTranslation
 import edu.artic.db.models.audioFile
+import edu.artic.localization.LanguageSelector
 import edu.artic.viewmodel.BaseViewModel
 import io.reactivex.subjects.BehaviorSubject
 import io.reactivex.subjects.Subject
 import javax.inject.Inject
 
 /**
+ * This is where the [AudioDetailsFragment] gets its information about a specific
+ * [ArticObject]. One such object may contain an [ArticAudioFile][edu.artic.db.models.ArticAudioFile],
+ * and if it does so we populate fields in this class with that as a primary
+ * source.
+ *
  * @author Sameer Dhakal (Fuzz)
+ * @see AudioTranslation
  */
 class AudioDetailsViewModel @Inject constructor() : BaseViewModel() {
     val title: Subject<String> = BehaviorSubject.create()
     val image: Subject<String> = BehaviorSubject.create()
+    val availableTranslations: Subject<List<AudioTranslation>> = BehaviorSubject.create()
+    val chosenTranslation: BehaviorSubject<AudioTranslation> = BehaviorSubject.create()
```

Comment:
consistency: `chosenTranslation: Subject<..>`

Reply:
Just to confirm: at the point where we need that internal value we should cast it, right?

Reply:
I dont think we need casting right? since its just a single type.

Reply:
we should use a default value and use the RX helpers `Optional` to ensure it always emits